routetable/testing_shims.go

```diff
@@ -17,60 +17,130 @@ package routetable
 import (
```

Comment:
Agreed `dataplane_shims.go`?  I'm now thinking we should shim the Handle and move the reconnect logic into the wrapper code.